crates/strategy/zksync/src/backend/runner.rs

```diff
@@ -125,6 +129,59 @@ impl BackendStrategyRunner for ZksyncBackendStrategyRunner {
             &ZksyncMergeState { persistent_immutable_keys: &ctx.persistent_immutable_keys };
         ZksyncBackendMerge::merge_zk_account_data(addr, active, fork_db, zk_state);
     }
+
+    fn transact_from_tx(
+        &self,
+        back: &mut Backend,
+        data: Bytes,
```

Comment:
We can pass a reference here instead since we only decide it. Optically a byte slice can be used perhaps.